src/main/java/com/google/firebase/database/core/Repo.java

```diff
@@ -67,14 +68,14 @@
 
   private static final String TRANSACTION_TOO_MANY_RETRIES = "maxretries";
   private static final String TRANSACTION_OVERRIDE_BY_SET = "overriddenBySet";
+
+  private static final Logger operationLogger = LoggerFactory.getLogger(Repo.class);
```

Comment:
You are now using this in places were you previously used 'transactionLogger'. I'd suggest naming this just 'logger'.

Reply:
Done